portfolio/src/main/webapp/js/main.js

```diff
@@ -36,21 +36,38 @@ function getComments() {
 /** Creates an element that represents a comment, including its delete button. */
 function createCommentElement(comment) {
   const commentElement = document.createElement('li');
-  commentElement.className = 'comment';
+  commentElement.className = 'row align-items-center';
 
-  const messageElement = document.createElement('span');
+  const profileElement = document.createElement('i');
+  profileElement.className = 'material-icons account col-lg-1 col-md-1 col-sm-1';
+  profileElement.innerText = 'account_circle';
+
+  const messageBoxElement = document.createElement('div');
+  messageBoxElement.className = 'col-lg-10 col-md-10 col-sm-10'
+  const nameElement = document.createElement('p');
+  nameElement.className = 'name';
+  nameElement.innerText = comment.name;
+  const messageElement = document.createElement('p');
+  messageElement.className = 'message';
   messageElement.innerText = comment.message;
+  messageBoxElement.appendChild(nameElement);
+  messageBoxElement.appendChild(messageElement);
 
   const deleteButtonElement = document.createElement('button');
-  deleteButtonElement.innerText = 'Delete';
+  const deleteIconElement = document.createElement('i')
+  deleteIconElement.className = 'material-icons delete col-lg-1 col-md-1 col-sm-1'
+  deleteIconElement.innerText = 'close';
```

Comment:
Consider extracting some helper methods here, for example `createMessageBoxElement(comment)`, for better readability.

Reply:
done